index.html

```diff
@@ -182,43 +242,87 @@ <h1 id="t_about">About me</h1>
 
       <section id="more_about">
 
-          <ul id="style1" class="laguage">
-            <li class="align">Language
-              <ul id="list" class="none">
-                <li>Javascript</li>
-                <li>Ruby</li>
-                <li>Html</li>
-                <li>Css</li>
-              </ul>
-            </li>
-          </ul>
-        
-          <ul id="style2" class="frameworks">
-            <li class="align">Frameworks
-              <ul id="list" class="none">
-                <li>Bootstrap</li>
-                <li>Ruby on Rails</li>
-                <li>RSPec</li>
-                <li>CapyBara</li>
-                <li>Selenium</li>
-              </ul>
-            </li>
-          </ul>
+        <ul id="style1" class="laguage">
+          <li class="align">Language
+            <ul id="list" class="none">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
+          </li>
+        </ul>
 
-          <ul id="style3" class="skills">
-            <li class="align">Skills
-              <ul id="list" class="none">
-                <li clas>Codekit</li>
-                <li>Github</li>
-                <li>Codepen</li>
-                <li>Gitlab</li>
-                <li>Terminal</li>
-              </ul>
-            </li>
-          </ul>
+        <ul id="style2" class="frameworks">
+          <li class="align">Frameworks
+            <ul id="list" class="none">
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
+          </li>
+        </ul>
+
+        <ul id="style3" class="skills">
+          <li class="align">Skills
+            <ul id="list" class="none">
+              <li clas>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </li>
+        </ul>
       </section>
     </section>
 
+    <footer id="form_section">
+      <header>
+        <h1 id="last_header">I'm always interested in hearing about new projects, so if you'd like to chat please get in
+          touch.</h1>
+      </header>
```

Comment:
[OPTIONAL] Note: You can have several `<header>` elements in one HTML document. However, `<header>` cannot be placed within a `<footer>`, `<address>` or another `<header>` element. check [this link](https://www.w3schools.com/tags/tag_header.asp#:~:text=Note%3A%20You%20can%20have%20several%20%3Cheader%3E%20elements%20in%20one%20HTML%20document.%20However%2C%20%3Cheader%3E%20cannot%20be%20placed%20within%20a%20%3Cfooter%3E%2C%20%3Caddress%3E%20or%20another%20%3Cheader%3E%20element.) for further explaination

Reply:
@jerryowusu Thank you. I will read that information

Reply:
Fixed.